lib/views/help/privacy.cy.html.erb

```diff
@@ -1,22 +1,11 @@
 <% @title = "Eich preifatrwydd" %>
 <%= render :partial => 'sidebar' %>
+
 <div id="left_column_flip" class="left_column_flip">
+  <%= render partial: 'translation_disclaimer' %>
+
   <h1 id="privacy"><%= @title %> <a href="#privacy">#</a> </h1>
   <dl>
-    <div class="changes">
-      <span style="font-size:16px">
-      <p>
-        <strong>Pennau i fyny!</strong> Rydym yn gwybod nad yw'r dudalen hon mor gyfoes ag y dymunwn. 
-        Rydyn ni'n ceisio datrys hyn; ond os na allwch ddod o hyd i'r hyn sydd ei angen arnoch yma, 
-        rhowch gynnig ar ein prif <a href="/help/privacy">Hysbysiad Preifatrwydd</a>, sydd yn Saesneg.
-      </p>
-      <p>
-        Os hoffech chi, gallwch weld cyfieithiad awtomataidd, drwy ddefnyddio
```

Comment:
It would be good to maintain this link, in some way - as it allows for an easy way to obtain the content in a language other than `en_GB`.

Reply:
Ah yeah okay ISWYM

Reply:
Added in 55fc54c